src/skin/css/firstRun.css

```diff
@@ -498,7 +498,6 @@ header .title-bar h1 {
   display:block;
   margin:2.5rem auto 2rem;
   padding-bottom:3rem;
-  width:94px;
```

Comment:
Why did we remove this?

Reply:
I removed to solve a UI bug with share links on narrow windows (see the PR description for a screenshot). Removing the 94px width fixed this UI bug and didn't introduce new bugs on all the screen sizes I checked, so I'm not sure why it was there